shuffle/app/(frontend)/polls/[slug]/admin/page.tsx

```diff
@@ -63,6 +65,24 @@ async function getData({ params }: PollAdminPageProps) {
     },
   });
 
+  const flaggedStatements = await prisma.flaggedStatement.findMany({
+    where: {
+      statementId: {
+        in: statements.map((statement) => statement.id),
+      },
+    },
+  });
+
+  const flaggedStatementsWithStatement = flaggedStatements.map((statement) => {
+    const statementWithStatement = {
+      ...statement,
+      statement: statements.find(
+        (s) => s.id === statement.statementId,
+      ) as Statement,
+    };
+    return statementWithStatement as FlaggedStatementsWithStatement;
```

Comment:
I think it would be more performant to use the include param of prisma instead of this.